src/test/scala/com/microsoft/azure/sqldb/spark/utils/SQLServerTestUtils.scala

```diff
@@ -0,0 +1,185 @@
+    val db = writeConfig.get[String](SqlDBConfig.DatabaseName).get
+    val properties = createConnectionProperties(writeConfig)
+    val user = writeConfig.get[String](SqlDBConfig.User).get
+    val password = writeConfig.get[String](SqlDBConfig.Password).get
+    val port = writeConfig.get[String](SqlDBConfig.PortNumber).getOrElse("1433")
+    if(db.equals(null) || url.equals(null)){
+      return null
+    }
+    Class.forName(DRIVER_NAME)
+    var dbConnect = ""
+    if(connectToDB){
+      dbConnect = ";database="+db
+    }
+    var conn: Connection = DriverManager.getConnection(createJDBCUrl(url, Some(port))+ dbConnect, user, password)
+    return conn
+  }
+
+  private def preFlightChecks(config: Map[String, String]): Boolean = {
+    var writeConfig = Config(config)
+    val conn = getConnection(writeConfig, false)
+    val dbName = writeConfig.get[String](SqlDBConfig.DatabaseName).get
+    if(conn.equals(null)){
+      return false
+    }
+
+    //Check if Server and Database Exist
+    var sql = "SELECT name from master.dbo.sysdatabases where name='"+ dbName + "' OR [name]='"+ dbName + "'"
+    try{
+      val stmt = conn.createStatement()
+      val req = stmt.executeQuery(sql)
+      conn.close()
+      var res = req.getString(1)
+      if(res.eq(null) || res.eq("") || !res.eq(dbName)){
+        return false
+      }
+      return true
+    } catch {
+      case e: Exception => return false
+    }
+  }
+
+  def createTable(config: Map[String, String], columns: Map[String, String]): Boolean ={
+    var writeConfig = Config(config)
+    val table = writeConfig.get[String](SqlDBConfig.DBTable).getOrElse(null)
+    if(table.equals(null)) {
+      return false
+    }
+    val conn = getConnection(writeConfig, true)
+    if(conn.equals(null)){
+      return false
+    }
+    if(!dataTypeVerify(columns)){
+      return false
+    }
+
+    var tableDroppedIfExists = dropTable(config)
+    if(tableDroppedIfExists){
+      var columnDef = ""
+      for(column <- columns){
+        columnDef += column._1 + " " + column._2 + ", "
+      }
+      var sql = "CREATE TABLE " + table + "(" + columnDef.substring(0, columnDef.length-1) + ");"
+      try{
+        val stmt = conn.createStatement()
+        stmt.executeUpdate(sql)
+        conn.close()
+      } catch {
+        case e: Exception => return false
+      }
+    } else {
+      return false
+    }
+
+    return true
+  }
+
+  def dropTable(config: Map[String, String]): Boolean ={
+    var writeConfig = Config(config)
+    val table = writeConfig.get[String](SqlDBConfig.DBTable).getOrElse(null)
+    if(table.equals(null)) {
+      return false
+    }
+    val conn = getConnection(writeConfig, true)
+    if(conn.equals(null)){
+      return false
+    }
+
+    // var sql = "IF EXISTS(SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = '" + table + "') DROP TABLE " + table + ";"
+    var sql = "DROP TABLE IF EXISTS " + table   //Supported SQL Server 2016 and beyond
```

Comment:
given this in a test, I will let this fly. otherwise in general there are still quite a few customers who use older versions of SQL hence the first way (checking IF EXISTS) might be a better option, but that's fine. No need to change this, just FYI.